### PR TITLE
MRP species names (unathi, nian)

### PR DESCRIPTION
**Pull request by lzk228**
## About the PR

Change species names, look cl

## Why / Balance

lore + mrp

## Media

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl:
- tweak: Reptilian species renamed to Unathi
- tweak: Moth species renamed to Nian

### CODE REVIEW

**Comment by lzk228** [comment]
fuckshit it was supposed to be 30 grand pr, but i pressed the wrong button

**Comment by Errant-4** [comment]
What, no name for the "arachnids"?

**Comment by lzk228** [comment]
> What, no name for the "arachnids"?

soon tm

**Comment by regenpower** [comment]
there was a discussion in [29448](https://github.com/space-wizards/space-station-14/pull/29448) about how the ss14 lizards are different from unathi (smoller and more silly™️), and that wizden explicitly does not have species lore

**Comment by Hanzdegloker** [comment]
YES, FINALLY

Slimepeople and Arachnids just need proper names now. It shouldn't annoy me so much but the generic names just take me out of any rp.

**Comment by Winkarst-cpu** [comment]
> YES, FINALLY
> 
> Slimepeople and Arachnids just need proper names now. It shouldn't annoy me so much but the generic names just take me out of any rp.

Spiderpeople >:)

**Comment by Hanzdegloker** [comment]
> > YES, FINALLY
> > 
> > Slimepeople and Arachnids just need proper names now. It shouldn't annoy me so much but the generic names just take me out of any rp.
> 
> Spiderpeople >:)

I wish you a very immovable rod

**Comment by SlamBamActionman** [comment]
Can we get an explanation for why these names specifically? "Lore+mrp" tells me nothing, and the names themselves sound so generic they could be applied to any fantasy species ever.

**Comment by Hanzdegloker** [comment]
> Can we get an explanation for why these names specifically? "Lore+mrp" tells me nothing, and the names themselves sound so generic they could be applied to any fantasy species ever.

Unathi and Nian are both used on ss13 paradise station, polaris also uses Unathi iirc

**Comment by dffdff2423** [comment]
I really don't think we should blindly copy ss13 species names. If you change the names of the species to be more inline with ss13 people will assume that we are using the ss13 lore. I prefer the generic names because it permits people to create slight variations of the species to suit their characters better (eg. Kobolds).

**Comment by SlamBamActionman** [comment]
> > Can we get an explanation for why these names specifically? "Lore+mrp" tells me nothing, and the names themselves sound so generic they could be applied to any fantasy species ever.
> 
> Unathi and Nian are both used on ss13 paradise station, polaris also uses Unathi iirc

"SS13 uses it" is useful to know since it shows established usage, but it in itself not a reason. Why do they use these names?

**Comment by Djungelskog2** [comment]
It's probably to make the universe feel a bit more fleshed out
Imagine your a pioneer and you meet a whole new race and they introduce themselves as "lizard people"

Or if the reverse happened and we described ourselves as "flesh lizards"

**Comment by dffdff2423** [comment]
In my mind, "Lizard People" and "Moth People" are an over generalized form of a group of species. People are welcome to play/introduce themselves as Unathi and Nian, or they can introduce themselves as some other lizard or moth like species.

File: Resources/Locale/en-US/cargo/bounties.ftl [hunk 1]
```diff
@@ -122,7 +122,7 @@ bounty-description-chili = Today's the Centcomm Chili Cookoff, and, well, a few
 bounty-description-rollerskates = CentComm Security is proposing a new strategy for helping officers win foot pursuits. Send them a couple so they cna learn how bad an idea this is.
 bounty-description-bedsheet = Someone in Atmos keeps turning down the heater, and we're all shivering in our beds. Please send us some extra sheets to stay warm.
 bounty-description-bandana =  Bzzzt... Transmission from prison planet OC-1001: We're... reorganizing our command structure. Send us some bandanas so we can tell gan- I mean, departments apart.
-bounty-description-steak = The vegetarian cook is refusing to make us anything with meat, and the lizards are getting restless. Can you smuggle us a few steaks to keep them happy?
+bounty-description-steak = The vegetarian cook is refusing to make us anything with meat, and the unathis are getting restless. Can you smuggle us a few steaks to keep them happy?
```
Comment (Errant-4) [inline Resources/Locale/en-US/cargo/bounties.ftl:125]: unathi

File: Resources/Locale/en-US/paper/book-authorbooks.ftl [hunk 2]
```diff
@@ -44,17 +44,17 @@ book-text-sloth-clown-sss = Once upon a time, in a space station far, far away,
 
 book-text-sloth-clown-pz = Chuckles was performing at a new space station, and he was excited to bring his brand of joy and laughter to a new audience. But this time, he wasn't alone. He had brought his trusty sloth friend, Snuggles, along for the adventure.
       As soon as they arrived, Chuckles and Snuggles went exploring the space station. They came across a group of Diona children, who were sad and upset. Chuckles asked them what was wrong, and they told him that they had lost their favorite toy, a small stuffed animal named Twinkle.
-      Chuckles knew that he had to help. He and Snuggles searched the space station, looking high and low for Twinkle. They even asked the other species for help. The Lizards were too busy sunbathing, the Humans were too busy working, but the Slimes were happy to help.
+      Chuckles knew that he had to help. He and Snuggles searched the space station, looking high and low for Twinkle. They even asked the other species for help. The Unathis were too busy sunbathing, the Humans were too busy working, but the Slimes were happy to help.
```
Comment (Errant-4) [inline Resources/Locale/en-US/paper/book-authorbooks.ftl:47]: unathi

File: Resources/Locale/en-US/paper/book-authorbooks.ftl [hunk 3]
```diff
@@ -44,17 +44,17 @@ book-text-sloth-clown-sss = Once upon a time, in a space station far, far away,
...
 
-book-text-sloth-clown-mmd = Chuckles was traveling through space with his friend Snuggles the sloth, looking for their next adventure. They came across a planet inhabited by Lizards, who were known for their love of games and challenges. Chuckles and Snuggles decided to explore the planet and see what kind of games they could find.
-      As they walked through the Lizards' city, they saw a crowd of Lizards gathered around a large game board. Chuckles and Snuggles went over to investigate, and they saw that the Lizards were playing a game called "The Maze of Mystery."
+book-text-sloth-clown-mmd = Chuckles was traveling through space with his friend Snuggles the sloth, looking for their next adventure. They came across a planet inhabited by Unathis, who were known for their love of games and challenges. Chuckles and Snuggles decided to explore the planet and see what kind of games they could find.
```
Comment (Errant-4) [inline Resources/Locale/en-US/paper/book-authorbooks.ftl:54]: unathi

File: Resources/Locale/en-US/paper/book-authorbooks.ftl [hunk 4]
```diff
@@ -44,17 +44,17 @@ book-text-sloth-clown-sss = Once upon a time, in a space station far, far away,
...
+      As they walked through the Unathis' city, they saw a crowd of Unathis gathered around a large game board. Chuckles and Snuggles went over to investigate, and they saw that the Unathis were playing a game called "The Maze of Mystery."
```
Comment (Errant-4) [inline Resources/Locale/en-US/paper/book-authorbooks.ftl:55]: unathi

File: Resources/Locale/en-US/tips.ftl [hunk 5]
```diff
@@ -81,7 +81,7 @@ tips-dataset-80 = As a Scientist, you can utilize upgraded versions of machines
 tips-dataset-81 = As a Scientist, you can build cyborgs using positronic brains and a chassis, they are just as useful as a new crew member.
 tips-dataset-82 = As a Medical Doctor, try to be wary of overdosing your patients, especially if someone else has already been on the scene. Overdoses are often lethal to patients in crit!
 tips-dataset-83 = As a Medical Doctor, don't underestimate your cryo pods! They heal almost every type of damage, making them very useful when you are overloaded or need to heal someone in a pinch.
-tips-dataset-84 = As a Medical Doctor, exercise caution when putting reptilians in cryopods. They will take a lot of extra cold damage, but you can mitigate this with some burn medicine or leporazine.
+tips-dataset-84 = As a Medical Doctor, exercise caution when putting unathis in cryopods. They will take a lot of extra cold damage, but you can mitigate this with some burn medicine or leporazine.
```
Comment (Errant-4) [inline Resources/Locale/en-US/tips.ftl:84]: unathi

File: Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml [hunk 6]
```diff
@@ -468,7 +468,7 @@
   parent: PosterBase
   id: PosterContrabandTheBigGasTruth
   name: "The Big Gas Giant Truth"
-  description: "Don't believe everything you see on a poster, patriots. All the lizards at central command don't want to answer this SIMPLE QUESTION: WHERE IS THE GAS MINER MINING FROM, CENTCOM?"
+  description: "Don't believe everything you see on a poster, patriots. All the unathis at central command don't want to answer this SIMPLE QUESTION: WHERE IS THE GAS MINER MINING FROM, CENTCOM?"
```
Comment (Errant-4) [inline Resources/Prototypes/Entities/Structures/Wallmounts/Signs/posters.yml:471]: unathi

File: Resources/ServerInfo/Guidebook/Mobs/Reptilian.xml [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <Document>
-  # Reptilians
+  # Unathis
```
Comment (Errant-4) [inline Resources/ServerInfo/Guidebook/Mobs/Reptilian.xml:2]: Unathi

File: Resources/ServerInfo/Guidebook/NewPlayer/YourFirstCharacter.xml [hunk 8]
```diff
@@ -42,9 +42,9 @@ Don't feel obligated to use any of these, though.
 Markings range from [color=#a4885c]scars and snouts[/color] to [color=#a4885c]tattoos and tails[/color].
 Think of them like [color=#a4885c]accessories[/color] that are a part of your body, for giving your character some extra flair and uniqueness. They cannot be changed during the round.
 
-Some species have more marking options than others. Humans are more limited in terms of markings than moths or reptilians.
+Some species have more marking options than others. Humans are more limited in terms of markings than nians or unathis.
```
Comment (Errant-4) [inline Resources/ServerInfo/Guidebook/NewPlayer/YourFirstCharacter.xml:45]: unathi

File: Resources/ServerInfo/Guidebook/NewPlayer/YourFirstCharacter.xml [hunk 9]
```diff
@@ -42,9 +42,9 @@ Don't feel obligated to use any of these, though.
...
 
-You can have a set number of markings in each '[color=#a4885c]category[/color]', as seen at the top. For example, reptilians can have one chest marking but two markings on the top of their heads.
+You can have a set number of markings in each '[color=#a4885c]category[/color]', as seen at the top. For example, unathis can have one chest marking but two markings on the top of their heads.
```
Comment (Errant-4) [inline Resources/ServerInfo/Guidebook/NewPlayer/YourFirstCharacter.xml:47]: unathi

**Review by Errant-4 — COMMENTED** [review]
I'm 90% sure that the plural of unathi is supposed to also be unathi

**Comment by Kukutis96513** [comment]
what's a unathi?..

**Comment by Thurtik** [comment]
> In my mind, "Lizard People" and "Moth People" are an over generalized form of a group of species. People are welcome to play/introduce themselves as Unathi and Nian, or they can introduce themselves as some other lizard or moth like species.

Yeah they're not called "reptilians" or "moth people" because they're canonically _called_ that, it's because that's what they _literally_ are

If people see this as a consistency issue I'd rather rename dionas and voxes to "plantae" or "plant people" and voxes to "avians" more than anything (but _please_ don't actually do that, the names are fine the way they are)

**Comment by deathride58** [comment]
> Can we get an explanation for why these names specifically?

These two names both come directly from SS13 servers.

The name "Unathi" comes from Baystation from early 2013. Specifically, it was a rebrand of "Soghun", which in turn was a fleshing-out (mostly lore-wise) of TG's lizards with the goal of turning them into proper aliens (rather than just being plain ol' lizardpeople). Unathi are fairly common across HRP servers, though some MRP servers opt to label their lizards "unathi" despite lacking any of the actual traits commonly associated with 'em. Unathi themselves have diverged pretty heavily from their roots as being renamed lizards (they even have a unique bodytype on most HRP servers!), to the point where it's simply naive (for lack of better word) to treat unathi and lizard as interchangeable species names.

The name "Nian" is lifted specifically from Paradise, and is a name that not many other SS13 servers use. Moths aren't a common sight on HRP servers in SS13 (as moths are a species that originate from NRP and LRP servers), and the HRP servers that do feature 'em all call 'em different names.

**Comment by SlamBamActionman** [comment]
I'm mostly interested in what etymologic meaning the names Nian and Unathi have. Diona are a pretty blatant reference to the Dionaea, the genus for the Venus Flytrap, while I assume Vox is related to the latin word vox meaning "voice" and how they screech like all hell.

**Comment by SlamBamActionman** [comment]
From a cursory google search, Nian is the name of a four-legged lion/dragon beast related to the Chinese New Year, and Unathi is a first name used in Africa.

**Comment by LankLTE** [comment]
Thank you for the PR, but maintainers have decided this isn't desirable for upstream. 
The changed species are intentionally generic. By keeping species as simply "lizard", the opportunity to RP out a lot of different things from, just a lizard, to a salamander, to a snake- whereas calling them "Unathi" comes with a lot of pre-existing lore and concepts from 13 that we don't tend to want here, and you're still free to RP out your character specifically being one. 
Diona and Vox have more specific names because, as a species, they're less generic. If we wanted to make other species like that, it'd need more effort than just changing the name.

**Comment by regenpower** [comment]
silly lizors rejoice!

**Pull request closed without merging**